### PR TITLE
A/B test to remove the "6 for 6" Guardian Weekly offer

### PR DESCRIPTION
**Pull request by KaliedaRik**
## What are you doing in this PR?
This PR sets up an A/B test for the suppression of the Guardian Weekly magazine '6 for 6 issues' offer.

These changes affect the copy presented on two Support site pages: Subscriptions, and Guardian Weekly. They apply across all regions (uk, us, ca, au, eu, int, nz).

+ __Change 1:__ On the Subscriptions page, for the variant, do not display the yellow-highlighted "6 issues for £6" copy
+ __Change 2:__ On the Guardian Weekly page, for the variant, change the copy appearing in the Hero roundel to: "Save up to 34% a year"
+ __Change 3:__ On the Guardian Weekly page in the first subscriber bullpoint, for the variant, change the copy to "Every issue delivered with up to 34% off the cover price" (was: 35%)
+ __Change 4:__ On the Guardian Weekly page in the Subscribe to the Guardian Weekly today section, remove the "6 for 6" card, including its CTA button

[**Trello Card**](https://trello.com/c/zVURGAWV/152-gw-removing-6-6-ab-test)

## Why are you doing this?
See this [Google Slides set](https://docs.google.com/presentation/d/1VPpsiphJA1VnxCaWo0omrkL3hrKkOZmAhrXP5LvvmiU/edit#slide=id.gf624b79472_0_0) for more details of the requirement. From the hypothesis given there:

_6/6 has been a strong offer, however contributes to high churn. By having such a deeply discounted offer that is always on makes it difficult to test alternative offers, or use 6/6 in discrete channels as a flex strategy. This test is to learn what is the net impact of removing this offer versus new BAU monthly pricing._

## Screenshots

Subscriptions page, control
us-subs-control

Subscriptions page, variant
us-subs-variant

Guardian Weekly page, control
us-weekly-control

Guardian weekly page, variant
us-weekly-variant

### CODE REVIEW

File: support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx [hunk 1]
```diff
@@ -109,16 +111,24 @@ type WeeklyProductPricesProps = {
...
 	productPrices,
 	orderIsAGift,
+	participations,
 }: WeeklyProductPricesProps): Product[] => {
 	const billingPeriodsToUse = orderIsAGift
 		? weeklyGiftBillingPeriods
-		: weeklyBillingPeriods;
+		: weeklyBillingPeriods(participations.sixForSixSuppression !== 'variant');
+
+	if (participations.sixForSixSuppression === 'variant') {
```
Comment (tomrf1) [inline support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx:127]: this block can be removed now

File: support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx [hunk 2]
```diff
@@ -132,15 +142,23 @@ const getProducts = ({
 					fixedTerm: false,
 					currency: 'GBP' as IsoCurrency,
 			  };
+
 		return weeklyProductProps(billingPeriod, productPrice, orderIsAGift);
 	});
 };
 
-function WeeklyProductPrices({
+// const WeeklyHero: React.FC<PropTypes> = ({
```
Comment (tomrf1) [inline support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx:150]: delete?

File: support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx (hunk 1 above)
Comment (KaliedaRik) [reply support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx:127]: Agreed - done

File: support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx (hunk 2 above)
Comment (KaliedaRik) [reply support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx:150]: Yes

File: support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx [hunk 3]
```diff
@@ -109,16 +111,25 @@ type WeeklyProductPricesProps = {
...
 
 const getProducts = ({
 	countryId,
 	productPrices,
 	orderIsAGift,
+	participations,
 }: WeeklyProductPricesProps): Product[] => {
 	const billingPeriodsToUse = orderIsAGift
 		? weeklyGiftBillingPeriods
 		: weeklyBillingPeriods;
```
[the remark this answers is not in the document]
Comment (KaliedaRik) [reply support-frontend/assets/pages/weekly-subscription-landing/components/weeklyProductPrices.tsx]: Fixed via PR #3318

**Review by tomrf1 — APPROVED** [review]
great! Let's wait for the signal from Marketing before merging this

**Review by rupertbates — APPROVED** [review]
Looks good, the only suggestion I'd make is that rather than passing the `participations` object around, it would be neater to extract a `suppressSixForSix` boolean at the top level and pass that down into the components. Happy to leave it as it is for now though as this is a test.

**Pull request merged**